internal/spdxlicense/license_list.go

```diff
@@ -109,18 +109,18 @@ var licenseIDs = map[string]string{
 	"bsd-1.0-clause":                         "BSD-1-Clause",
 	"bsd-1.0.0-clause":                       "BSD-1-Clause",
 	"bsd-2-clause":                           "BSD-2-Clause",
-	"bsd-2-clause-freebsd":                   "BSD-2-Clause-FreeBSD",
-	"bsd-2-clause-netbsd":                    "BSD-2-Clause-NetBSD",
+	"bsd-2-clause-freebsd":                   "BSD-2-Clause-Views",
+	"bsd-2-clause-netbsd":                    "BSD-2-Clause",
```

Comment:
I don't think we should be changing the semantics of `freebsd` and `netbsd` in this way and below.

Reply:
Yes, we should.

This is explained at:
- https://spdx.org/licenses/BSD-2-Clause-FreeBSD.html
- https://spdx.org/licenses/BSD-2-Clause-NetBSD.html

Using BSD-2-Clause-FreeBSD or BSD-2-Clause-NetBSD is invalid SPDX and is reported as such by https://tools.spdx.org/app/validate/